matter_server/server/device_controller.py

```diff
@@ -337,8 +350,8 @@ def attribute_updated_callback(
             new_value = transaction.GetAttribute(path)
             LOGGER.debug("attribute updated -- %s - new value: %s", path, new_value)
```

Comment:
should this log include node ID ?

Reply:
in fact that whole log line can be removed, I used it for debugging as I couldn't use a breakpoint here due to that callback called by the SDK

Reply:
Adjusted the logging a bit so the node events get logged to their own log child. Figured that logging might still come in handy